server/db/queries/getQuestions.js

```diff
@@ -1,5 +1,5 @@
 const connection = require('../config/connection');
 
-const getQuestions = () => connection.query('SELECT Name, title__c, wordsLimit__c, tips__c, expectations__c FROM  salesforce.questions__c')
+const getQuestions = () => connection.query('SELECT title__c, wordsLimit__c, tips__c, expectations__c FROM  salesforce.questions__c')
```

Comment:
We need to get the Name from salesforce to be an id, then this id is used as a key when we render TabBody component. This way we get rid of some warnings in the console!

Reply:
ahh cool I wasn't sure about it, I'll add it to the test database